toolkit/pallets/block-production-log/src/test.rs

```diff
@@ -0,0 +1,39 @@
+use super::*;
+use frame_support::traits::UnfilteredDispatchable;
+use mock::*;
+use sp_consensus_slots::Slot;
+use sp_runtime::BoundedVec;
+
+#[test]
+fn first_append_should_succeed() {
+	new_test_ext().execute_with(|| {
+		let call = Call::<Test>::append { id: make_id::<Test>(1) };
+		call.dispatch_bypass_filter(RuntimeOrigin::none()).unwrap();
+
+		assert_eq!(Log::<Test>::get().to_vec(), vec![(Slot::from(1001000), make_id::<Test>(1))]);
+	})
+}
+
+#[test]
+fn append_to_end_of_log() {
+	new_test_ext().execute_with(|| {
+		let existing_item = vec![(Slot::from(100), make_id::<Test>(1))];
+		Log::<Test>::put(
+			BoundedVec::<_, <mock::Test as pallet::Config>::MaxLogSize>::truncate_from(
+				existing_item,
+			),
+		);
+
+		let call = Call::<Test>::append { id: make_id::<Test>(2) };
+		call.dispatch_bypass_filter(RuntimeOrigin::none()).unwrap();
```

Comment:
consider call without bypass of the filter
```suggestion
		assert_ok!(BlockProductionLog::append(RuntimeOrigin::none(), make_id::<Test>(2)));
```